<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BullsAndCows</class>
 <widget class="QMainWindow" name="BullsAndCows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BullsAndCows</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(154, 175, 207);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="m_pB_records">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background-color: #FFFFFF;
 border: 2px solid #696969;
 border-radius: 7px;
 padding: 5px;
}

QPushButton:pressed {
 background-color: #A9A9A9;
 border: 2px solid #696969;
 border-radius: 7px;
 padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Рекорды</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="m_label_top">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 11pt &quot;Segoe UI&quot;;
 //background-color: rgba(255, 255, 255, 0.3);
 //border: 2px solid rgba(255, 255, 255, 0.3);
 border-radius: 7px;
 padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Добро пожаловать!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="m_pB_Attempt">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background-color: #FFFFFF;
 border: 2px solid #696969;
 border-radius: 7px;
 padding: 5px;
}

QPushButton:pressed {
 background-color: #A9A9A9;
 border: 2px solid #696969;
 border-radius: 7px;
 padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Проверить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="m_pB_NewGame">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
 background-color: #FFFFFF;
 border: 2px solid #696969;
 border-radius: 7px;
 padding: 5px;
}

QPushButton:pressed {
 background-color: #A9A9A9;
 border: 2px solid #696969;
 border-radius: 7px;
 padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Новая игра</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	font: 11pt &quot;Segoe UI&quot;;
 //background-color: rgba(255, 255, 255, 0.3);
 //border: 2px solid rgba(255, 255, 255, 0.3);
 border-radius: 7px;
 padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Введите число:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="m_lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
 background-color: #FFFFFF;
 border: 2px solid #696969;
 border-radius: 7px;
 padding: 5px;
}</string>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTableWidget" name="m_tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
    background-color: #FFFFFF;
    padding: 4px;
    font-size: 10pt;
    border-style: none;
    border-bottom: 1px solid rgb(0, 0, 0);
    border-right: 1px solid rgb(0, 0, 0);
}

QHeaderView::section:horizontal
{
    border-top: 1px solid rgb(0, 0, 0);
}

QHeaderView::section:vertical
{
    border-left: 1px solid rgb(0, 0, 0);
}

QTableWidget QTableCornerButton::section {
    background-color: #FFFFFF;
    border: 1px solid rgb(0, 0, 0);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>114</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Число</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Быки</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Коровы</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
